backend/app/assets/stylesheets/spree/backend/components/_navigation.scss

```diff
@@ -69,40 +133,54 @@ nav.menu {
 
     &.selected {
       background: $color-navbar-active-bg;
+      @include media($desktop) {
+        position: relative;
+      }
 
-      > a {
-        color: $color-navbar-active;
+      &:not(:hover) > ul {
+        @include media($desktop) {
+          display: none;
+        }
+        @include media($tablet) {
+          display: block;
+        }
       }
-    }
 
-    &.selected li {
-      padding-left: 2em;
-    }
+      > ul {
+        @include media($desktop) {
+          @include flyout;
```

Comment:
IMO all breakpoints can profit from a flyout. Is there a particular reason you hide them from small viewports?